SVGPortability.h

```diff
@@ -2,8 +2,12 @@
 
 #if TARGET_OS_IPHONE
 #   import <UIKit/UIKit.h>
-#   define SVGUI(kls) UI##kls
+#   define View UIView
```

Comment:
If we want to do it this way, `@compatibility_alias` is more appropriate. Also, we need to use our prefix, we'd be very likely to conflict with a host project otherwise.